index.html

```diff
@@ -35,5 +35,118 @@ <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
         <a href="#middle" id="down-button" class="fa-solid fa-chevron-down"></a>
       </header>
     </section>
+
+    <main id="middle" class="font-style">
+      <h3>My Recent Works</h3>
+      <div class="container">
+        <article class="grid-item">
+          <img src="https://placehold.jp/350x250.png" alt="Placeholder Image">
+          <h5>Multi-Post Stories <span>Gain+Glory</span></h5>
+          <ul class="list-format list-container">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button class="project-button" type="button" name="button">See project</button>
+        </article>
+        <article class="grid-item">
+          <img src="https://placehold.jp/350x250.png" alt="Placeholder Image">
+          <h5>Multi-Post Stories <span>Gain+Glory</span></h5>
+          <ul class="list-format list-container">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button class="project-button" type="button" name="button">See project</button>
+        </article>
```

Comment:
- [ ] Your project card does not match the Figma designs. The tech stack portion does not have the appropriate width or flex. Please take a look at the image below for comparison, you can see the difference. Kindly make your design match the Figma design. 
- [ ] The project card title should not be that small, kindly use `h2` or `h3` for the title instead of `h5`.
- [ ] Please use `flexbox` to align the content of each card. This means that the element article with class `grid-item` should use flexbox to align its content. 

Your design | Figma Design
---- | ----
<img width="302" alt="image" src="https://github.com/flov3rh/portfolio/assets/84156714/3c370e10-c3e7-43de-ad18-14c0bed9fe3c"> | <img width="245" alt="image" src="https://github.com/flov3rh/portfolio/assets/84156714/6fa9b506-044b-45f8-9782-47f34e63fd8c">